src/include/gpmc/dialogs.py

```diff
@@ -19,7 +19,7 @@
 from adcommon.creds import YCreds, switch_domains
 from adcommon.ui import CreateMenu, DeleteButtonBox
 
-ENABLE_EXPERIMENTAL = @ENABLE_EXPERIMENTAL@
+ENABLE_EXPERIMENTAL = bool(os.getenv("YAST2_GPMC_EXPERIMENTAL"))
```

Comment:
This doesn't maintain the current behavior. All you've done is add an environment variable to enable the experimental features, whereas before these were enabled by default on openSUSE, and disabled on SLE.